x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/action/apikey/TransportGrantApiKeyAction.java

```diff
@@ -30,7 +30,7 @@
  */
 public final class TransportGrantApiKeyAction extends TransportGrantAction<GrantApiKeyRequest, CreateApiKeyResponse> {
 
-    private final ApiKeyGenerator generator;
+    private final ApiKeyManager apiKeyManager;
```

Comment:
Decidedly not in love with `Manager`. The other alternative I have is `ApiKeyWriter` -- it's a little more specific which is good but `Writer` has a lot of `XContent` connotation which is bad.